iconservice/iconscore/icon_pre_validator.py

```diff
@@ -259,6 +261,14 @@ def _check_balance(self, context: 'IconScoreContext', from_: 'Address', value: i
     def _is_inactive_score(self, address: 'Address') -> bool:
         is_contract = address.is_contract
         is_zero_score_address = address == ZERO_SCORE_ADDRESS
-        is_score_active = self._deploy_storage.is_score_active(None, address)
+        is_score_active = self._is_score_active(address)
         _is_inactive_score = is_contract and not is_zero_score_address and not is_score_active
         return _is_inactive_score
+
+    def _is_score_active(self, address: 'Address') -> bool:
```

Comment:
we don't need to use this function.
we have to determine remove it or not

Reply:
IconPreValidator class refactoring will go on in another branch later.